recipe/meta.yaml

```diff
@@ -27,19 +27,19 @@ requirements:
 
 test:
   imports:
-    - coverage
+    - pytest_httpserver
   source_files:
     - tests
   requires:
     - pip
     - pytest >=7.1.3
-    - pytest-cov
     - requests >=2.28.1
     - types-requests >=2.28.9
-    - pre-commit 3.2.1
     - mypy
     - toml
-    - types-toml >=0.10.8.6
+    - packaging
```

Comment:
The tests would not complete without `packaging` and `execptiongroup` and claimed that `pytest 7.3.1` required them.

Reply:
`exceptiongroup` shouldn't be needed for python 3.11

Reply:
This is concerning though. These packages are part of the `pytest` run dependencies and should have been pulled in through pytest.

Reply:
I agree with Marco, it's quite concerning.

Reply:
`conda search -i pytest=7.3.1` says:
```
pytest 7.3.1 py38hca03da5_0
---------------------------
dependencies: 
  - attrs >=19.2.0
  - exceptiongroup >=1.0.0
  - iniconfig
  - packaging
  - pluggy >=0.12,<2.0
  - python >=3.8,<3.9.0a0
  - tomli >=1.0.0

pytest 7.3.1 py39hca03da5_0
---------------------------
dependencies: 
  - attrs >=19.2.0
  - exceptiongroup >=1.0.0
  - iniconfig
  - packaging
  - pluggy >=0.12,<2.0
  - python >=3.9,<3.10.0a0
  - tomli >=1.0.0

pytest 7.3.1 py310hca03da5_0
----------------------------
dependencies: 
  - attrs >=19.2.0
  - exceptiongroup >=1.0.0
  - iniconfig
  - packaging
  - pluggy >=0.12,<2.0
  - python >=3.10,<3.11.0a0
  - tomli >=1.0.0

pytest 7.3.1 py311hca03da5_0
----------------------------
dependencies: 
  - attrs >=19.2.0
  - iniconfig
  - packaging
  - pluggy >=0.12,<2.0
  - python >=3.11,<3.12.0a0
  - tomli >=1.0.0
```

To me pytest is all good and we need to take a look at why you need packaging and exceptiongroup here.

Reply:
This is what I get when I pull `exceptiongroup` and `packaging`
```
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
export PREFIX=/Users/smartin/work/build_ci/pytest-httpserver_1684504037762/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
export SRC_DIR=/Users/smartin/work/build_ci/pytest-httpserver_1684504037762/test_tmp
import: 'pytest_httpserver'
import: 'pytest_httpserver'
+ python -m pip check
pytest 7.3.1 requires exceptiongroup, which is not installed.
pytest 7.3.1 requires packaging, which is not installed.
Tests failed for pytest-httpserver-1.0.6-py38hca03da5_0.tar.bz2 - moving package to /Users/smartin/work/build_ci/broken
WARNING:conda_build.build:Tests failed for pytest-httpserver-1.0.6-py38hca03da5_0.tar.bz2 - moving package to /Users/smartin/work/build_ci/broken
TESTS FAILED: pytest-httpserver-1.0.6-py38hca03da5_0.tar.bz2
```